app/helpers/application_helper.rb

```diff
@@ -38,7 +38,7 @@ def markdown(description)
       underline: true,
       highlight: true,
       quote: true,
-      footnotes: true
+      footnotes: true,
```

Comment:
you dont really need the coma if its the last value, but its a good practice.

Reply:
yeah I only just noticed that at the end. good spot :-)